### PR TITLE
Remove redundant backend route

### PR DESCRIPTION
**Pull request by mjauvin**
I tested this on nginx and apache on both Laravel 9 and Laravel 11 by making sure the route cache was cleared.

No issues so far.

### CODE REVIEW

**Comment by LukeTowers** [comment]
@mjauvin does this still work if backendUri is set to an empty string? `''`

**Comment by mjauvin** [comment]
> @mjauvin does this still work if backendUri is set to an empty string? `''`

Yes, that works too.

File: modules/backend/routes.php [hunk 1]
```diff
@@ -20,7 +20,7 @@
     Event::fire('backend.beforeRoute');
 
     /*
-     * Other pages
+     * Entry point
```
Comment (bennothommo) [inline modules/backend/routes.php]: Let's describe what is actually happening so it's clear
```suggestion
     * Route everything to `Backend\Classes\BackendController`, which itself acts as a router for the Backend.
```

**Comment by mjauvin** [comment]
@LukeTowers care to merge this so we can test it in develop at a larger scale before we release 1.2.8 ?

**Pull request merged**